src/calculated-property-rule.ts

```diff
@@ -108,31 +108,36 @@ export class CalculatedPropertyRule extends Rule {
 
 		// modify list properties to match the calculated value instead of overwriting the property
 		if (this.property.isList) {
-			// re-calculate the list values
-			var newList = newValue;
+			if (!this.property.isInited(obj)) {
+				Property$init(this.property, obj, newValue);
+			}
```

Comment:
This clause prevents us from running the "list update" logic if the property is not yet initialized.

Reply:
Makes sense to me! Was this part of the fix, or just a related optimization?

Reply:
Not part of the fix, just a bug I discovered when writing tests.

Reply:
👍